packages/vuelidate/src/core.js

```diff
@@ -476,22 +477,25 @@ export function setValidations ({
    */
 
   const $model = key ? computed({
-    get: () => unwrap(state[key]),
+    get: () => unwrap(unwrap(state)[key]),
```

Comment:
This extra unwrap is because of the computed above, right?

Reply:
Yes, the state can be a `ref` because of the `computed` above, or because the user passes a `ref`.